chain/app/db_app.go

```diff
@@ -25,6 +29,7 @@ func NewDBBandApp(
 ) *dbBandApp {
 	app := NewBandApp(logger, db, traceStore, loadLatest, invCheckPeriod, baseAppOptions...)
 
+	dbBand.ZoracleKeeper = app.ZoracleKeeper
```

Comment:
Oh god.... Do we really need to do this for all keepers?

Reply:
:(

Reply:
If we don't work like this how we send keeper to handleMsg